cwd/app.js

```diff
@@ -337,8 +340,13 @@
   const renderView = view => {
     console.log(`Rendering view: ${view.name}`);
 
+    // Removes highlight from previous gauge if any
+    gauge = document.getElementById(`gauge-${prevGauge + 1}`);
```

Comment:
Always define a new variable as `const` if you can, and I think you can here.  Unless it is first defined above?